MapboxNavigation/DefaultStyle.swift

```diff
@@ -3,17 +3,23 @@ import Foundation
 
 extension UIColor {
     class var defaultRouteCasing: UIColor { get { return .defaultTintStroke } }
-    class var defaultRouteLayer: UIColor { get { return UIColor.defaultTintStroke.withAlphaComponent(0.6) } }
+    class var defaultRouteLayer: UIColor { get { return #colorLiteral(red:0.00, green:0.70, blue:0.99, alpha:1.0) } }
```

Comment:
no traffic it defaults to the blue unknown/low color:

![image](https://user-images.githubusercontent.com/1058624/28222043-9c2706f6-687a-11e7-8314-ed0632e8bb17.png)